libs/styles/styles.css

```diff
@@ -124,6 +124,13 @@
   --icon-size-s: 32px;
   --icon-size-xs: 24px;
   --icon-size-xxs: 16px;
+
+    /* border-radius sizes */
+    --border-radius-s: 4px;
+    --border-radius-l: 8px;
+    --border-radius-xl: 16px;
+    --full-border-radius: 50%;
```

Comment:
Are we certain authors understand that this would create a circle only if the element has an aspect ratio of 1:1? Here's a quick example of what I mean - [link](https://jsbin.com/nekizuhuvi/1/edit?html,css,output). I think this might be misleading if not properly explained.

Reply:
This was the assumption we has taken that the authors would give images with aspect ratio 1:1 only. Because if we apply aspect-ratio property to the image, it may get a little distorted and authors may not be able to debug it.